server_config_parser.cc

```diff
@@ -18,13 +18,21 @@ std::vector<std::unique_ptr<http::handler> >& handlers_out) {
     for (size_t i = 0; i < config.statements.size(); i++) {
         // Add echo handler with its base URL
         if (config.statements[i]->tokens[0] == "echo") {
-            handlers_out.push_back(std::move(std::unique_ptr<http::handler>(
-                new http::handler_echo(config.statements[i]->tokens[1]))));
+            // Form should be: echo /base_url
+            if (config.statements[i]->tokens.size() == 2 && 
```

Comment:
Although it is not too complex, the 2 should be a constant variable with an appropriate name.